modules/ROOT/pages/_partials/identity-management-demo.adoc

```diff
@@ -10,7 +12,8 @@ From the showcase app:
 Cordova::
 +
 --
-. Login on app start.
+. To log in, click the *Login* button on the first page.
```

Comment:
I can't remember if it will be actually on `the first page`. Maybe just not say that?